src/diffusers/schedulers/scheduling_pndm.py

```diff
@@ -335,8 +355,8 @@ def _get_prev_sample(self, sample, timestep, timestep_prev, model_output):
         # sample -> x_t
         # model_output -> e_θ(x_t, t)
         # prev_sample -> x_(t−δ)
-        alpha_prod_t = self.alphas_cumprod[timestep + 1 - self._offset]
-        alpha_prod_t_prev = self.alphas_cumprod[timestep_prev + 1 - self._offset]
```

Comment:
@patrickvonplaten this change breaks this test:

https://github.com/huggingface/diffusers/blob/f4781a0b27ffb3ea61ecd25b0b87305e0960304e/tests/test_scheduler.py#L633-L655

However, I think the test must be wrong, because there we use the default `offset=0`, which means that we are using `self.alphas_cumprod[1]` for the timestep `0`, which doesn't make sense. I assume the value in the test is computed using the current implementation, rather than a reference value from elsewhere?

Reply:
Hmm, I think the test breaks because `set_alpha_to_one` defaults to `True` no?

Reply:
I kept the change and adjusted the test value.